.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
 
 script:
   - make test
-  - make test_scispacy
+  # - make test_scispacy
```

Comment:
you don't need scispacy anymore?

Reply:
I think a later commit restores it but if not I will amend. The reason for removing is in the PR description.